src/export/digitalPlanning/model.ts

```diff
@@ -133,14 +143,60 @@ export class DigitalPlanning {
     };
   }
 
+  mapPassportToPreApplicationPayload(): PreApplicationPayload {
+    return {
+      data: {
+        user: {
+          role: this.passport.data?.["user.role"]?.[0],
+        },
+        applicant:
+          this.passport.data?.["user.role"]?.[0] === "applicant"
+            ? this.getApplicant()
+            : this.getApplicantWithAgent(),
+        property: this.getProperty(),
+        application: {
+          type: {
+            value: "preApp",
+            description: "Pre-application",
+          },
+          fee: {
+            payable: this.passport.data?.["application.fee.payable"] as number,
+            // Account for self-pay (has passport data) or invite to pay (has govUkPayment only)
+            ...((this.passport.data?.["application.fee.reference.govPay"] ||
+              this.govUkPayment) && {
+              reference: {
+                govPay:
+                  this.passport.data?.["application.fee.reference.govPay"]?.[
+                    "payment_id"
+                  ] || this.govUkPayment?.payment_id,
+              },
+            }),
+          },
+          declaration: this.getApplicationDeclaration(),
+          information: this.getPreAppInformation(),
+        },
+        proposal: {
+          description: this.passport.data?.["proposal.description"] as string,
+          boundary: this.getProposedBoundary(),
+        },
+      },
+      responses: this.getResponses(),
+      files: this.getFiles(),
+      metadata: this.getMetadata() as PlanXMetadata,
+    };
+  }
```

Comment:
Similar to comments above, I'm introducing a new `mapPassportToPreApplicationPayload()` within the _existing_ class here to share/re-use as many of the existing mappings as possible.

`this.payload` and `validatePayload()` are the only other pieces that need to be application-type aware.